build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+	id 'java'
+}
+
+group = 'com.controller'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```

Comment:
Clean Gradle dependencies:
Rewrite all dependencies with one standart: if it is double quotes then all dependencies should look like it: 
```
implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```
if you are using label names:
```
compile group: 'javax.persistence', name: 'javax.persistence-api', version: '2.2'
```

Also, remove comment links for the maven repository, it is not interesting information
`// https://mvnrepository.com/artifact/javax.persistence/javax.persistence-api`